src/explorer-components/PageCard/index.js

```diff
@@ -25,16 +25,19 @@ class Header extends React.Component {
 
 class Content extends React.Component {
   render() {
-    const { children, className, bodyClass } = this.props;
+    const { children, className, bodyClass, noBanner } = this.props;
+    console.log('nobanner', noBanner);
```

Comment:
lets talk about this - I had a different envisioning vs. just doing two with a noBanner.